superset/viz.py

```diff
@@ -935,7 +946,10 @@ def to_series(self, df, classed='', title_suffix=''):
             ys = series[name]
             if df[name].dtype.kind not in 'biufc':
                 continue
-            series_title = name
+            if isinstance(name, (list, tuple)):
+                series_title = [str(title) for title in name]
```

Comment:
does that affect bar chart sorting?

Reply:
It should not since we are treating the title as string in the JS anyways.